src/events/index.spec.js

```diff
@@ -239,6 +239,113 @@ describe('Events', function() {
     });
   });
 
+  describe('getEventTypesByClient', function() {
+    context('the clientId is provided', function() {
+      let clientId = faker.random.uuid();
+      let promise;
+      let request;
+      let eventTypeFromServer;
+
+      beforeEach(function() {
+        eventTypeFromServer = fixture.build('eventType', { fromServer: true });
+
+        request = {
+          ...baseRequest,
+          get: this.sandbox.stub().resolves(eventTypeFromServer)
+        };
+
+        const events = new Events(baseSdk, request);
+        events._baseUrl = expectedHost;
+
+        promise = events.getEventTypesByClient(clientId);
+      });
+
+      it('gets the eventType from the server', function() {
+        expect(request.get).to.be.calledWith(
+          `${expectedHost}/clients/${clientId}/types`
+        );
+      });
+
+      it('returns the requested event type', function() {
+        return expect(promise).to.be.fulfilled.and.to.eventually.deep.equal(
+          eventTypeFromServer
+        );
+      });
+    });
+
+    context('the clientId is not provided', function() {
+      it('throws an error', function() {
+        const events = new Events(baseSdk, baseRequest);
+        const promise = events.getEventTypesByClient();
+
+        return expect(promise).to.be.rejectedWith(
+          'A client ID is required for getting types'
+        );
+      });
+    });
+  });
+
+  describe('getEventsByType', function() {
+    let typeId = faker.random.uuid();
+    let facilityId = faker.random.uuid();
+    let promise;
+    let request;
+    let eventsFromServer;
+
+    beforeEach(function() {
+      eventsFromServer = times(
+        fixture.build('event', { fromServer: true }),
```

Comment:
`eventsFromServer = fixture.buildList('event', 10, null, { fromServer: true });`

(even better, use `faker.random.number({ min: 5, max: 20 })` instead of `10`)